influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -1004,10 +1000,10 @@ mod tests {
             "+-----+----------------------+",
             "| 5.0 | 1970-01-01T00:05:00Z |",
             "| 6.0 | 1970-01-01T00:05:30Z |",
-            "| 1.0 | 1970-01-01T00:00:10Z |",
-            "| 2.0 | 1970-01-01T00:01:05Z |",
-            "| 3.0 | 1970-01-01T00:02:27Z |",
             "| 4.0 | 1970-01-01T00:04:10Z |",
+            "| 3.0 | 1970-01-01T00:02:27Z |",
+            "| 2.0 | 1970-01-01T00:01:05Z |",
+            "| 1.0 | 1970-01-01T00:00:10Z |",
             "+-----+----------------------+",
         ];
         let actual = get_table_batches(&write_buffer, "foo", "cpu", &ctx).await;
```

Comment:
I can't comment on the next line but instead of using `assert_batches_eq!`, try `assert_batches_sorted_eq!` which should guarantee the order every time and keep the result consistent for CI etc.